details/operator-sdk/controllers/connector.go

```diff
@@ -138,6 +141,17 @@ func (r *AstraConnectorController) deployConnector(ctx context.Context,
 		natsSyncClientStatus.Status = UnregisterNSClient
 	}
 
+	// if we are registered and have a clusterid let's set up the asup cr
```

Comment:
@jharrod does this look right? using clusterID instead of cloudID to easily identify what cluster this belongs too